modules/ROOT/pages/resource-matchers-reference.adoc

```diff
@@ -0,0 +1,93 @@
+= Resource Matchers
+
+== getResourceAsStream(String)
+
+Returns the resource from the path String as a stream.
+
+.Example
+[source,xml,linenums]
+----
+<set-payload
+  value="#[MunitTools::getResourceAsStream('util/files/example.txt')]"/>
+----
+
+== getResourceAsReusableStream(String)
+
+Returns the resource from the path String as a reusable stream.
+
+.Example
+[source,xml,linenums]
+----
+ <set-payload
+   value="#[MunitTools::getResourceAsReusableStream('util/files/example.txt')]"/>
+----
+
+== getResourceAsString(String)
+
+Returns the resource from the path String as a String.
+
+.Example
+[source,xml,linenums]
+----
+<set-payload
+  value="#[MunitTools::getResourceAsString('util/files/example.txt')]"/>
+----
+
+You can also specify the encoding:
+
+.Example
+[source,xml,linenums]
+----
+<set-payload
+  value="#[MunitTools::getResourceAsString('util/files/exampleISO.txt', 'ISO-8859-1')]"/>
+----
+
+== getResourceAsByteArray(String)
+
+Returns the resource from the path String as a byte array.
+
+.Example
+[source,xml,linenums]
+----
+ <set-payload
+   value="#[MunitTools::getResourceAsByteArray('util/files/example.txt')]"/>
+----
+
+== queueSize(String)
+
+Returns the size of the queue specified inside `#[MunitTools::queueSize()]`.
+
+For example, to assert the size of a queue named `QUEUE_SIZE`:
+.Example
```

Comment:
I think there's a missing new line here, .Example shows in line 60 in the preview